docs/commands.md

```diff
@@ -0,0 +1,486 @@
+---
+title: Commands
+---
+
+The driver comes with a set of many available commands, in addition to the commands included in the
```

Comment:
I assume we might also add geolocation commands there. 
Endpoints are: https://github.com/appium/appium/blob/5aaca9b11cc293c0ab0b87c88f03222f30893141/packages/base-driver/lib/protocol/routes.js#L379

Reply:
There seems to be a lot of other routes that aren't documented here, but for now I've only added the geolocation ones.